opensoundscape/torch/models/cnn.py

```diff
@@ -823,33 +812,31 @@ def predict(
                 - if None, does not log to wandb
 
         Returns:
-            scores: df of post-activation_layer scores
+            df of post-activation_layer scores
             predictions: df of 0/1 preds for each class
             unsafe_samples: list of samples that failed to preprocess
```

Comment:
The .predict method now only returns the df of post-activation_layer scores, so I these two lines should be removed from the docstring.